shared/README.md

```diff
@@ -0,0 +1,126 @@
+
+MDS uses Version 1 UUIDs.
+
+## Costs and currencies
+
+Fields specifying a monetary cost use a currency as specified in [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217#Active_codes). All costs should be given as integers in the currency's smallest unit. As an example, to represent $1 USD, specify an amount of `100` (100 cents).
+
+If the currency field is null, USD cents is implied.
+
+## Devices
+
+MDS defines the *device* as the unit that transmits GPS or GNSS signals for a particular vehicle. A given device must have a UUID (`device_id` below) that is unique within the Provider's fleet.
+
+Additionally, `device_id` must remain constant for the device's lifetime of service, regardless of the vehicle components that house the device.
+
+## Vehicle Types
+
+The list of allowed `vehicle_type` values in MDS is:
+
+| `vehicle_type` | Description |
+|--------------| --- |
+| bicycle      | Anything with pedals, including recumbents; can include powered assist |
+| car          | Any automobile |
+| scooter      | Any motorized mobility device intended for one rider |
+| moped        | A motorcycle/bicycle hybrid that can be powered or pedaled |
+
+## Propulsion Types
+
+| `propulsion`      | Description                                            |
+| ----------------- | ------------------------------------------------------ |
+| `human`           | Pedal or foot propulsion                               |
+| `electric_assist` | Provides power only alongside human propulsion         |
+| `electric`        | Contains throttle mode with a battery-powered motor    |
+| `combustion`      | Contains throttle mode with a gas engine-powered motor |
+
+A vehicle may have one or more values from the `propulsion`, depending on the number of modes of operation. For example, a scooter that can be powered by foot or by electric motor would have the `propulsion` represented by the array `['human', 'electric']`. A bicycle with pedal-assist would have the `propulsion` represented by the array `['human', 'electric_assist']` if it can also be operated as a traditional bicycle.
+
+## Vehicle States
+
+This table describes the list of vehicle conditions that may be used by regulators to assess the disposition of individual vehicles and fleets of vehicles.  Some of these states describe vehicles in the Public Right-of-Way (PROW), and others represent vehicles that are not.  One state (`unknown`) implies that PROW status is unknown.
+
+In a multi-jurisdiction environment, the status of a vehicle is per-jurisdiction.  For example, a vehicle may be in the `trip` status for a county that contains five cities, and also in the `trip` status for one of those cities, but `elsewhere` for the other four cities.  In such a condition, generally a Provider would send the device data to the over-arching jurisdiction (the county) and the vehicle state with respect to each city would be determined by the Agency managing the jurisdictions.
+
+| `vehicle_state` | Description |
```

Comment:
Suggestion on this table: break out an "In PROW" column to make that distinction more consistent and explicit for each state.

Reply:
done